Resources/doc/reference/installation.rst

```diff
@@ -55,10 +56,15 @@ You will need to configure SonataArticleBundle's entities, if you plan to use th
                 article:  Application\Sonata\ArticleBundle\Entity\Article
                 fragment: Application\Sonata\ArticleBundle\Entity\Fragment
 
-            fragments: # this is currently in development, hope we'll get this soon :)
+            fragments:
                 sonata.article.fragment.title:
                 sonata.article.fragment.text:
 
+.. note::
+
+    We plan to improve the fragments available to allow a configuration for each article type.
+    For example, you will want the fragment 'Comments' only on articles of type 'Blog'.
+
```

Comment:
> Since this bundle is still a work in progress, have BC Break later will not be an issue for the moment ? 

It won't be if we don't make a release. Imagine there is release 1.0.0 . You can make as many bc-breaks as you want if you fix them before next 1.* release. The problem with my first option is not forward-compatible, as you outlined.
But maybe you could make it work for this configuration and hardcode `simple_array_provider`: 

```
fragment_whitelist_provider:
    simple_array_provider:
        - sonata.article.fragment.comment
        - sonata.article.fragment.whatever    
```

That way you are still forward-compatible with something more complicated later, and you are not forced to implement `simple_array_provider` and so on right now.